website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and click **Create**. Select **Shared Signals Framework Provider** and continue with the following form values:
+
+| Field           | Value                        |
+| --------------- | ---------------------------- |
+| Name            | `Apple Business Manager SSF` |
+| Signing Key     | `[Your Signing Key]`         |
+| Event Retention | `days=30`                    |
+
+Click **Finish** and confirm that the new SSF provider is listed in the overview.
+
+:::tip A Blank SSF Config URL is expected
+
+Keep in mind the **SSF Config URL** will be blank until the SSF provider is assigned to an application as a backchannel provider. We'll return to collect this URL after creating the application.
+
+:::
+
+### 5. Create application
+
+From the authentik Admin interface, navigate to **Applications** -> **Applications** and click **Create**. Continue with the following form values:
+
+| Field                | Value                      |
+| -------------------- | -------------------------- |
+| Name                 | Apple Business Manager     |
+| Slug                 | `abm`                      |
+| Provider             | Apple Business Manager     |
+| Backchannel Provider | Apple Business Manager SSF |
+
+Click **Create** and confirm that the application is listed in the overview page.
+
+:::tip Your SSF Config URL is now available
+
+Assigning an SSF provider to a application will generate **SSF Config URL**.
+
+Navigate to **Providers** -> **Apple Business Manager SSF** and copy the `SSF Config URL` value to your text editor.
+
+:::
+
+---
+
+## Configuration checklist
+
+Let's recap the steps you've taken to configure authentik as an identity provider for Apple Business Manager:
+
+1. Created scope mappings:
+    - [x] User's profile information
+    - [x] read access
+    - [x] Management access.
+2. Created a OIDC provider to handle the authentication flow.
+    - [x] Added the scope mappings to the provider, **as well as the default `profile` scope.**
+    - [x] Specified the `Redirect URIs/Origins` and `Signing Key`.
+    - [x] Noted the `Client ID`, `Client Secret`, and `OpenID Configuration URL` in your text editor.
+3. Created a Shared Signals Framework provider to handle the backchannel communication between authentik and Apple Business Manager.
+    - [x] Specified the `Signing Key` and `Event Retention`.
+4. Created an application and assigned the SSF provider as a backchannel provider.
+    - [x] Noted the `SSF Config URL` in your text editor.
+
+:::tip Port numbers required
+
+By default, authentik will generate the **SSF Config URL** and **OpenID Configuration URL** without the default HTTPS port (443). However, Apple requires the port number to be included when providing the URLs in the configuration.
+
+From your text editor, update the URLs to include the port number `443`:
```

Comment:
?? Is there a command that they need to run in their...  by text editor do you mean their VSCode or whatever IDE they use?